index.html

```diff
@@ -211,10 +211,24 @@
           <h2>LDP Containers</h2>
           <p>
             Implementations MUST support the creation and management of [[!LDP]] Containers.
-            LDP Direct Containers MUST NOT permit <code>ldp:contains</code> as their membership-predicate
-            and requests that would do so MUST fail with 409 Conflict.
+          </p>
+          <p>
+            LDP Containers MUST distinguish containment triples from membership and minimal-container triples.
+            If an implementation cannot make this distinction solely from the predicate, it MUST NOT permit
```

Comment:
> If an implementation cannot make this distinction solely from the predicate

For example, are you saying here that if `ldp:contains` is used as a membership predicate in some cases (maybe because it is being used as an `ldp:hasMemberRelation`), then the distinction of whether `ldp:contains` is a containment triple or a membership triple based on the predicate alone is not possible... and therefore, `ldp:contains` can not actually be used as a membership predicate??